### DISCUSSION TITLE
Scores overlay while live streaming

### DISCUSSION THREAD

**Discussion by DemonViewLLC** [gopro/labs#633] [Feature Requests]
Just seen (once again) someone streaming a sports game and wanting to put a score tracker on their footage.  Since they are only going from the camera to the app for livestreaming this isn't an option  (they would have to go the OBS route).  Now I'm thinking, GoPro labs with an overlay could possibly do this. So question time!
Would it be possible to add a basic score template overlay while live streaming? 
Could this be added to the QR app as an extra.  So for simplification, a tab with 2 score boxes and the user enters in a score to the relevant box, this updates the QR code, user then holds it up in front of the camera to update the overlay while live streaming.
This means that people could broadcast their local games straight from the camera without having to go the OBS route. Just a thought.

**Comment by dnewman-gpsw** [comment]
I just tried it and is works now.

Demo video : https://www.twitch.tv/videos/1970442193

To make this work first scan (once)    !MQRDR=1  (this enables QR detection while recording, harmless it is my default.)

Live stream start via Labs or through Quik (it shouldn't matter, I used Labs LS start in my tests.) 

Then scan your score (live)

oMBURN="(0,41)[BC]Tigers 3 : 2 Pandas"  <- team names and scores here

(x,y) is the offset from the screen position
[BC} is screen position BC - Bottom Center, TC - Top Center, ML - Middle Left, MR - Middle Right, TR - Top Right etc.

**Reply by DemonViewLLC** [reply]
Excellent stuff, I'll let the OP on Reddit know, I may even make this a tutorial for a forthcoming video.  IMHO GoPro Labs is something that gives the GoPro a huge advantage over the competition.  Good stuff!

**Reply by HammerJD** [reply]
What did you use to generate the score QR? I think I am not putting the info in the overlay page from GoPro labs incorrectly. Or are you using a different QR app in the demo video?

**Reply by HammerJD** [reply]
nevermind...found the QR Control app!

**Reply by lsmith77** [reply]
did that tutorial ever get published?

**Reply by lsmith77** [reply]
FYI I created two tickets related to this #750 + #751
